roles/rsyslog/templates/output_elasticsearch.j2

```diff
@@ -37,25 +37,44 @@ ruleset(name="{{ item.name }}") {
             serverport="{{ item.server_port | d(9200) | int }}"
             template="{{ item.template | d("es_template") }}"
             searchIndex="{{ item.searchIndex | d("index_template") }}"
-            dynSearchIndex="{{ item.dynSearchIndex | d("on") }}"
```

Comment:
```
    dynSearchIndex="{{ item.dynSearchIndex | d(true) | ternary('on', 'off') }}"
```
https://docs.ansible.com/ansible/2.8/user_guide/playbooks_filters.html#id8 - `ternary` - this filter is provided by Ansible since Ansible 2.8, no need to worry about jinja version

Reply:
Thank you, @richm. Updated.